support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYou.jsx

```diff
@@ -176,27 +178,36 @@ const ContributionThankYou = ({
       createReferralCodes={campaignSettings && campaignSettings.createReferralCodes}
       campaignCode={campaignSettings && campaignSettings.campaignCode}
     />,
-    shouldShow: true,
+    shouldShow: !isEnvironmentMoment,
   };
   const ausMapAction = {
     component: <ContributionThankYouAusMap />,
     shouldShow: countryId === 'AU',
   };
+  const articleShareAction = {
+    component: <ContributionThankYouArticleShare
+      header="Share Guardian journalism"
+      body="Help extend the reach of our environmental reporting by sharing it with your friends, family and followers."
+    />,
+    shouldShow: isEnvironmentMoment,
+  };
 
   const defaultActions = [
     signUpAction,
     signInAction,
     marketingConsentAction,
+    socialShareAction,
```

Comment:
I think this should be at the end of the list so it appears at the bottom of the second column

Reply:
Hm, I assumed it was meant to go after `marketingConsentAction`, as that's where it appears in the design on Figma. I'll run this by Hyuna/Iona/Victoria on chat.

https://www.figma.com/file/rgmpdsf5GHWYZBQSn9Ad8b/Climate-Moment-2020?node-id=1166%3A12240

Reply:
sorry yeah - just the socialShareAction, not the new article share one

Reply:
Ah, gotcha. I've changed it now ✅